spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/LocalDeployerAutoConfiguration.java

```diff
@@ -35,13 +35,15 @@
 @AutoConfigureOrder(Ordered.HIGHEST_PRECEDENCE)
 public class LocalDeployerAutoConfiguration {
 
-	@Bean
+	public static final String DEPLOYER = "Local";
+
+	@Bean(DEPLOYER + "AppDeployer")
 	@ConditionalOnMissingBean(AppDeployer.class)
 	public AppDeployer appDeployer(LocalDeployerProperties properties) {
```

Comment:
Should we consider using the method name for the bean name but adding a `@Qualifier` value of simply "local"? I guess it depends on the access pattern for these beans.